### PR TITLE
Added /use token

### PR DESCRIPTION
**Pull request by zek7rom**

### CODE REVIEW

**Comment by HoeenCoder** [comment]
@zek7rom can you fix the travis issues?

**Comment by Lord-Haji** [comment]
@HoeenCoder @zek7rom  Travis issues were resolved

File: spacialgaze-plugins/misc.js [hunk 1]
```diff
@@ -327,4 +327,52 @@ exports.commands = {
...
+		case 'avatar':
+			msg = '|html|<center>' + user.name + ' has redeemed a avatar token.<br/><img src="' + target[1] + '" alt="avatar"/><br/>';
+			msg += '<button class="button" name="send" value="/customavatar set ' + user.userid + ', ' + target[1] + '">Apply Avatar</button></center>';
+			delete user.tokens[target[0]];
+			return SG.messageSeniorStaff(msg);
+		case 'declare':
+			msg += '|html|<center>' + user.name + 'has redeemed a global declare token.<br/> Message: ' + target[1] + "<br/>";
+			msg += '<button class="button" name="send" value=/globaldeclare ' + target[1] + '">Globally Declare the Message</button></center>';
+			delete user.tokens[target[0]];
+			return SG.messageSeniorStaff(msg);
```
Comment (HoeenCoder) [inline spacialgaze-plugins/misc.js:350]: Messages aren't being sent as parsed HTML.

`[13:41:20] ~SG Server: |html|<center>HoeenHerohas redeemed a global declare token.<br/> Message: Hello World!<br/><button class="button" name="send" value=/globaldeclare Hello World!">Globally Declare the Message</button></center>`

Also your missing a space between `usernamehas`
(this applies to all the cases in the switch statement, not just this one)

File: spacialgaze-plugins/misc.js [hunk 2]
```diff
@@ -327,4 +327,52 @@ exports.commands = {
...
+			if (!target[2]) return this.errorReply('/usetoken emote, [name], [img]');
+			msg += '|html|<center>' + user.name + 'has redeem a emote token.<br/><img src="' + target[1] + '" alt="emote"/><br/>';
+			msg += '<button class="button" name="send" value="/emote add ' + target[1] + ', ' + target[2] + '">Set emote</button></center>';
+			delete user.token[target[0]];
+			return SG.messageSeniorStaff(msg);
+		default:
+			return this.errorReply('An error occured in the command.'); // This should never happen.
+		}
+	},
+	usetokenhelp: ['/usetoken [token], [info] - Redeem a token from the shop. Accepts the following arguments: '],
```
Comment (HoeenCoder) [inline spacialgaze-plugins/misc.js]: Help info never got finished.

File: spacialgaze-plugins/misc.js (hunk 1 above)
Comment (Lord-Haji) [reply spacialgaze-plugins/misc.js:350]: Anyways @zek7rom shall do this my only job was to fix Travis issues

**Pull request merged**